packages/plugins/flow/resolvers/src/visitor.ts

```diff
@@ -55,9 +55,7 @@ export class FlowResolversVisitor extends BaseResolversVisitor<RawResolversConfi
   }
 
   protected formatRootResolver(schemaTypeName: string, resolverType: string, declarationKind: DeclarationKind): string {
-    return `${schemaTypeName}?: ${resolverType}${resolverType.includes('<') ? '' : '<>'}${this.getPunctuation(
-      declarationKind
-    )}`;
+    return `${schemaTypeName}?: ${resolverType}${this.getPunctuation(declarationKind)}`;
```

Comment:
The `<>` is a syntax error in every case, especially in the case of Enums that won't take any generic arguments.